scripts/download_IMG_files.py

```diff
@@ -0,0 +1,63 @@
+"""
+This script downloads IMG files from the specified S3 bucket to the local directory in NERSC.
+"""
+
+
+import os
+from collections import defaultdict
+from pathlib import Path
+
+import boto3
+
+SECRET_KEY = os.environ.get('SECRET_KEY')
+ACCESS_KEY = 'cdm-admin'
+# Tunnel to the MinIO server - `ssh -f -N -L localhost:49002:ci07:9002 <kbase_dev_username>@login1.berkeley.kbase.us`
+ENDPOINT_URL = 'http://localhost:49002/'
+BUCKET = 'dts-staging'
+
+COLL_ROOT = Path('/global/cfs/cdirs/kbase/collections')
+SOURCE_DIR = COLL_ROOT / 'collectionssource' / 'NONE' / 'CDM' / 'IMG'
+
+FILE_SUFFIXES = ['.fna']  # ['.fna', '.gff', '.faa'] - '.gff' is also available to download. Potentially '.faa' files as well.
+
+
+def main():
+    s3_client = boto3.client('s3',
+                             endpoint_url=ENDPOINT_URL,
+                             aws_access_key_id=ACCESS_KEY,
+                             aws_secret_access_key=SECRET_KEY)
+
+    response = s3_client.list_objects_v2(Bucket=BUCKET)
```

Comment:
does this have a maximum number of objects returned?

Reply:
👍 realized it and fixed.

Reply:
:+1: